config/common.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+import (
+	"github.com/tron-us/go-btfs-common/protos/guard"
```

Comment:
i will quote what eric's first comment left for me:
we use first import portion as build-in library like 'strconv';
and second portion is library of project itself, in this project it's 'gp-common-btfs'
and third portion would be third-party library like 'go-common', 'zap'

Not sure if we are going to stick with this format or not.

Reply:
Yes, we should stick to it.

Reply:
Done the change